backend/config/config.yaml

```diff
@@ -8,7 +8,7 @@ springServer:
   shutdownTimeout: 30 #time in second
 
 artifactStoreConfig:
-  artifactStoreType: NFS #S3, GCP, NFS
+  artifactStoreType: NFS #S3, GCS, NFS
```

Comment:
should be added to config.default.yaml too

Reply:
Done and also added config for GCS.